src/stelnet/EmptyIntel.java

```diff
@@ -4,6 +4,10 @@
 
 public class EmptyIntel extends BaseIntelPlugin {
 
+    protected String getName() {
+        return "Empty Intel";
```

Comment:
Use L10n if you want to make this translation friendly.

Reply:
Something like `return L10n.get(ExplorationL10n.EMPTY_TITLE);`, add enum; probably should be in a generic one enum, not `ExplorationL10n`.

Reply:
Actually, `EmptyIntel` may not need `getName`, I need to double check.